cluster/addons/calico-policy-controller/calico-node-vertical-autoscaler-deployment.yaml

```diff
@@ -17,7 +17,6 @@ spec:
       labels:
         k8s-app: calico-node-autoscaler
       annotations:
-        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
Do we need to add anything to mark these pods as critical with the first-class api that replaced critical pod annotations?

Reply:
That makes sense, but I need some suggestions from folks who are familiar with these files.

Reply:
I just took a look at all the yaml files, they all have `priorityClass: system-node-critical` to set them to critical.